webapp/components/form/Dropdown/store/actions/getItemsDialog.js

```diff
@@ -2,7 +2,7 @@ import * as StringUtils from '@core/stringUtils'
 import { State } from '../state'
 
 // Utility function used to get or fetch the dialog items
-export const getItemsDialog = async ({ state, value }) => {
```

Comment:
items can be an async function and this function can return items() which is a promise. 
It is safer to leave async